### DISCUSSION TITLE
When is the next update dropping?

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [mswgen/oc-updater#21] [General]
About 2 months have passed, and I know that OpenCore Version 0.9.6 is just on the horizon. So, I hope that the next update for OpenCore Updater drops around the release of OC 0.9.6 or atleast giving it just support for 0.9.5 soon.

**Comment by dropdb220** [comment]
I was busy lately due to school exams.
The new version is being built now, so it'll be available in a hour.

**Comment by dropdb220** [comment]
https://github.com/mswgen/oc-updater/releases/tag/1.2.20
